### PR TITLE
Olwa/documentation

### PR DESCRIPTION
**Pull request by owalk**

### CODE REVIEW

File: docs/specSheet.md [hunk 1]
```diff
@@ -171,6 +173,11 @@ Any entities that can interact with each other will be listed on the entity map.
 
 Entities will know where they are on the entity map by storing their own location indivdually. Both the individual location and the entity map location should be updated on movement.
 
+### Level Generation
+Each map will be created manually in the Tiled map editor program.
+
+In the future, new maps will be automatically generated as they are needed for endless gameplay options. The algorythm to generate maps should be defined here at that time.
```
Comment (Sean-Nishi) [inline docs/specSheet.md:179]: Spelling of algorithm

File: docs/specSheet.md [hunk 2]
```diff
@@ -209,16 +216,24 @@ Damsels will wander the map when spawned.
 * if Lunk is within 12 squares of a damsel, they will walk towards Lunk
 * if Lunk is within 5 squares of a damsel, they will stand still and make heart emojis at Lunk
 
-## Score System
-### Health Bar
+## HUD Elements
+
+The HUD will show the follow at all times of game play:
```
Comment (Sean-Nishi) [inline docs/specSheet.md:221]: Grammer, probably want "...will show the following...". Also, I thought gameplay was one word?

**Review by Sean-Nishi — CHANGES_REQUESTED** [review]
Looks good except for the two comments

File: docs/specSheet.md (hunk 1 above)
Comment (owalk) [reply docs/specSheet.md]: fixed

File: docs/specSheet.md (hunk 2 above)
Comment (owalk) [reply docs/specSheet.md]: this is fixed, and it definitely is just 1 word

**Review by Sean-Nishi — APPROVED** [review]
No comments. Looks good

**Pull request merged**